### PR TITLE
Fix for issue #4092: Remove unused field from Project model

### LINKED ISSUE #4092: Remove unused field from Project model
When removing the comments app from the codebase in https://github.com/rtfd/readthedocs.org/pull/3802. A field was missing to be removed, we need to remove this and make a migration.

https://github.com/rtfd/readthedocs.org/blob/696a25f9b3350bb047caf1899a876aa1aa0fcbaa/readthedocs/projects/models.py#L140-L143

### PR DESCRIPTION
**Pull request by Alig1493**
Close #4092

### CODE REVIEW

**Comment by Alig1493** [comment]
@stsewd @safwanrahman I'll look into it.

**Comment by Alig1493** [comment]
@stsewd @safwanrahman I've looked into it to the best of my ability. Would you like to take a look into it?

File: readthedocs/projects/migrations/0027_remove_unused_field.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    operations = [
+        migrations.RemoveField(
+            model_name='project',
+            name='allow_comments',
+        ),
+        migrations.RemoveField(
+            model_name='project',
+            name='comment_moderation',
+        ),
+        migrations.AlterField(
```
Comment (safwanrahman) [inline readthedocs/projects/migrations/0027_remove_unused_field.py]: Why this Altering?

Comment (stsewd) [reply readthedocs/projects/migrations/0027_remove_unused_field.py]: The rest of alters are from missing migrations (just changes in help text or options) https://github.com/rtfd/readthedocs.org/issues/4201

**Comment by safwanrahman** [comment]
Looks good. r+

**Comment by ericholscher** [comment]
Bumped the migration name, and will merge this :+1:

**Pull request merged**